OrbitQt/orbittreeview.cpp

```diff
@@ -2,6 +2,12 @@
 // Copyright Pierric Gimmig 2013-2017
 //-----------------------------------
 
+// This needs to be first because if it is not GL/glew.h
```

Comment:
Please use "OpenGl.h" instead of including glew.h manually.  OpenGl.h includes the opengl related headers in the correct order.

Reply:
Done.. had to remove Core.h include from OpenGl.h in order for this to work, but seems like this is right fix.